packages/x-components/src/x-modules/facets/components/lists/selected-filters-list.vue

```diff
@@ -69,6 +70,19 @@
     @Prop()
     protected facetId: Facet['id'] | undefined;
 
+    /**
+     * Kebab trans formation of the facet id.
+     *
+     * @param facetId - The id of the facet to transform in kebab case.
+     *
+     * @returns A kebab case id.
+     *
+     * @internal
+     */
+    protected slotName(facetId: string): string {
```

Comment:
Same as the facets, why not just map the item like in every component. In any case remember that this is not a valid method name for our code style guidelines, and the documentation is also wrong spelled. It is also usually better to document in an active voice: `Transforms the facet id to kebab case`, rather than `kebab transformation of the facet id`, which sounds weird when reading it.